src/pages/AddServers.js

```diff
@@ -150,11 +151,20 @@ class AddServers extends BaseUpdateWizardPage {
   // values, then set size is the same as the array list length,
   // otherwise the list contains duplicate values.
   hasDuplicates = (arrayList) => {
-    return (new Set(arrayList)).size !== arrayList.length;
+    if(isEmpty(arrayList)) {
+      return false;
+    }
+    // filter out empty items
+    let cleanList = arrayList.filter(item => !isEmpty(item));
+    if(isEmpty(cleanList)) {
```

Comment:
Again, remove this check. It is unnecessary.

Reply:
this is actual fix point where I can have array list contain like 2 empty item...if arrayList has [ undefined, undefined], cleanList is empty, so this check is necessary. This actually fixed the problem where has 2 server items , both of them have empty mac-addr...

Reply:
That doesn't make sense. If `cleanList` was empty, then the check on the next line (163) which compares the size of the set with the size of the list would still return false.